Folder API: Optionally force deleting Grafana 8 alerts when deleting a folder (or error) (#36427)
grafana/grafana: docs/sources/http_api/folder.md

`DELETE /api/folders/:uid`

> Deletes an existing folder identified by uid together with all dashboards (and their alerts) stored in the folder, if any. This operation cannot be reverted.

Reviewer: ```suggestion
Deletes an existing folder identified by UID along with all dashboards (and their alerts) stored in the folder. This operation cannot be reverted.
```

> If [Grafana 8 Alerts]({{< relref "../alerting/unified-alerting/_index.md" >}}) are enabled, you can set the optional query parameter `forceDeleteRules=false` so that requests will fail with 400 (Bad Request) if the folder contains any Grafana 8 Alerts. If this parameter is set to `true` then it will delete any Grafana 8 Alerts under this folder.

Reviewer: ```suggestion
If [Grafana 8 Alerts]({{< relref "../alerting/unified-alerting/_index.md" >}}) are enabled, you can set an optional query parameter `forceDeleteRules=false` so that requests will fail with 400 (Bad Request) error if the folder contains any Grafana 8 Alerts. However, if this parameter is set to `true` then it will delete any Grafana 8 Alerts under this folder.
```
